cpp/modules/luma.gl/core/src/glfw-animation-loop.cpp

```diff
@@ -103,15 +102,14 @@ auto GLFWAnimationLoop::_createDevice(const wgpu::BackendType backendType) -> st
   }
 
   WGPUDevice cDevice = backendAdapter.CreateDevice();
-  auto device = wgpu::Device::Acquire(cDevice);
-
-  return std::make_unique<wgpu::Device>(std::move(device));
+  return wgpu::Device{cDevice};
+  //  return wgpu::Device::Acquire(cDevice);
```

Comment:
Acquire was the problem? What does it do?

Reply:
It wasn't the Acquire, I left that change in there by accident while trying different things.

It was actually just the proc table not being set as early as it should. When passing the `Device` object, internal reference counting mechanism calls out to WebGPU function that's defined in the proc table, which wasn't set in the very beginning of the initialization, hence the misleading crash